README.md

```diff
@@ -200,4 +233,5 @@ be written to `web/controllers/api/v1/README.md`.
 * `:titles`: Allows you to specify explicit titles for some of your modules.
 For example `[{YourApp.Api.V1.UserController, "API /v1/users"}]` will
 change the title (Table of Contents entry and heading) for this controller.
+* `:prefix`: Allows you to remove the prefix of the test module names
```

Comment:
Where is this config used?

Reply:
Here: https://github.com/api-hogs/bureaucrat/pull/34/files#diff-d36f774419f37ac24b6ba39063685d10R144